{
	"id": "CVE-2021-29619",
	"vulnerabilities": [
		{
			"content": {
				"id": "CVE-2021-29619",
				"title": "Segfault in `tf.raw_ops.SparseCountSparseOutput`",
				"description": "TensorFlow is an end-to-end open source platform for machine learning. Passing invalid arguments (e.g., discovered via fuzzing) to `tf.raw_ops.SparseCountSparseOutput` results in segfault. The fix will be included in TensorFlow 2.5.0. We will also cherrypick this commit on TensorFlow 2.4.2, TensorFlow 2.3.3, TensorFlow 2.2.3 and TensorFlow 2.1.4, as these are also affected and still in supported range.",
				"severity": [
					{
						"type": "cvss_v31",
						"source": "GitHub_M",
						"cvss_v31": {
							"vector": "CVSS:3.1/AV:L/AC:H/PR:L/UI:N/S:U/C:N/I:N/A:L",
							"base_score": 2.5,
							"base_severity": "LOW",
							"temporal_score": 2.5,
							"temporal_severity": "LOW",
							"environmental_score": 2.5,
							"environmental_severity": "LOW"
						}
					}
				],
				"cwe": [
					{
						"source": "GitHub_M",
						"cwe": [
							"CWE-755"
						]
					}
				],
				"references": [
					{
						"source": "CVE",
						"url": "https://github.com/tensorflow/tensorflow/commit/82e6203221865de4008445b13c69b6826d2b28d9"
					},
					{
						"source": "CVE",
						"url": "https://github.com/tensorflow/tensorflow/security/advisories/GHSA-wvjw-p9f5-vq28"
					},
					{
						"source": "GitHub_M",
						"url": "https://github.com/tensorflow/tensorflow/commit/82e6203221865de4008445b13c69b6826d2b28d9"
					},
					{
						"source": "GitHub_M",
						"url": "https://github.com/tensorflow/tensorflow/security/advisories/GHSA-wvjw-p9f5-vq28"
					}
				],
				"modified": "2024-08-03T22:11:06.083Z"
			}
		}
	],
	"data_source": {
		"id": "mitre-v5",
		"raws": [
			"vuls-data-raw-mitre-v5/2021/CVE-2021-29619.json"
		]
	}
}
